elements/pfe-sass/maps/_general.scss

```diff
@@ -3,19 +3,26 @@ $pfe-vars: (
   container-padding:             16px,
   content-spacer:                24px,
 
-  font-size:                     16px,
-  line-height:                   1.5,
+  line-height--heading:          1.2,
+  line-height:                   1.3,
 
-  font-family:                   '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--heading:          '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--code:             '"Overpass Mono", Consolas, Monaco, 'Andale Mono', monospace',
+  font-family:                   '"Red Hat Text", "RedHatText", "Overpass", Overpass, Helvetica, Arial, sans-serif',
+  font-family--heading:          '"Red Hat Display", "RedHatDisplay", "Overpass", Overpass, Helvetica, Arial, sans-serif',
+  font-family--code:             '"Overpass Mono", Consolas, Monaco, "Andale Mono", monospace',
 
+  font-size--heading--xxl:       42px,
+  font-size--heading--xl:        38px,
   font-size--heading--alpha:     32px,
```

Comment:
@castastrophe I just realized that in the [redhat-theme typography](https://gitlab.corp.redhat.com/uxdd/redhat-theme/-/blob/5.1.0/dist/advanced-theme.css#L8) we are using the variables without the word "heading".

redhat-theme:  `--pfe-theme--font-weight--xl`
patternfly elements:  `--pfe-theme--font-size--heading--xl`

Should we create a backwards compatibility map in the redhat-theme and move fwd with the variable in PFE?

Reply:
+1